certification/internal/policy/podmanexec/validate_operator_bundle.go

```diff
@@ -0,0 +1,45 @@
+package podmanexec
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/komish/preflight/certification"
+)
+
+type ValidateOperatorBundlePolicy struct {
+}
+
+func (p ValidateOperatorBundlePolicy) Validate(bundle string) (bool, error) {
+	stdouterr, err := exec.Command("/usr/local/bin/operator-sdk", "bundle", "validate", "--verbose", bundle).CombinedOutput()
```

Comment:
I suggest using `which` command to find the full path of the `operator sdk` command. Using `/usr/local/bin/operator-sdk` as default is good.

Reply:
Good suggestion. Went ahead and added this.